### PR TITLE
add  "License :: OSI Approved :: NASA Open Source Agreement (NOSA)"

### PR DESCRIPTION
**Pull request by ebo**
Request to add a new NASA Open Source Agreement (NOSA) classifier:

     "License :: OSI Approved :: NASA Open Source Agreement (NOSA)"

According to various documentation[*], NOSA is an approved OSI open-source license, but is not contained in any of the online classifier lists.

Inclusion will help facilitate using NOSA in various python projects.

* https://opensource.gsfc.nasa.gov/nosa.php

### CODE REVIEW

File: src/trove_classifiers/__init__.py [hunk 1]
```diff
@@ -310,6 +310,7 @@
     "License :: OSI Approved :: Mozilla Public License 1.1 (MPL 1.1)",
     "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
     "License :: OSI Approved :: Mulan Permissive Software License v2 (MulanPSL-2.0)",
+    "License :: OSI Approved :: NASA Open Source Agreement (NOSA)",
```
Comment (di) [inline src/trove_classifiers/__init__.py:313]:
```suggestion
    "License :: OSI Approved :: NASA Open Source Agreement v1.3 (NASA-1.3)",
```
Per https://opensource.org/license/nasa1-3-php, OSI calls this "NASA Open Source Agreement v1.3" and the SPDX Identifier is `NASA-1.3`

**Review by di — CHANGES_REQUESTED** [review]

**Comment by ebo** [comment]
If this was already there and I missed it, I will feel rather stupid.  
Sorry for wasting everyone's time.

**Comment by tcaduser** [comment]
Hi @ebo, it looks to me like @di is just requesting you to accept the proposed change.  @di asked me to do the same thing for my proposed license addition.

**Comment by ebo** [comment]
Ah.  I misunderstood the point of the comment.  I would say that I 
approve the change (not sure where I need to do this, but it is OK with 
me).  I wish I had read this earlier in the day -- NASA's Open Source 
Initiative meeting was this afternoon, and I would have been able to get 
them to approve of the name or bump it up.  They were surprised that it 
was named NASA-{ver} instead of NOSA, but I can run this up the chain 
and talk to the lawyers who approve the licensing.

I would say for now go with NASA-{ver} as that is what OSI has, and I 
will talk to NASA's licensing lawyers to see if they are OK with that or 
if I should submit a name change through OSI, et. al.

**Comment by ebo** [comment]
do you want me to make these changes and resubmit the pull request?  I was confused on what you wanted me to do next.

**Comment by tcaduser** [comment]
I would try:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by di** [comment]
@ebo If the [change I made](https://github.com/pypa/trove-classifiers/pull/167/commits/1a76a74eb0722101dad108fd30f8d9d9ed1be6d8) is OK with you, we can merge this.

**Comment by ebo** [comment]
Yes, it is OK.  Sorry, I have been down with cluster headaches.  Only u 
for a few to try to eat something.

**Review by di — APPROVED** [review]
Thanks! Hope you feel better.

**Pull request merged**